tests/conftest.py

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/python3
+# Copyright 2019 Northern.tech AS
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        https://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+
+import pytest
+
+from mender_test_containers.container_props import *
+from mender_test_containers.conftest import *
+
+TEST_CONTAINER_LIST = [
+    MenderTestRaspbian,
+    #MenderTestQemux86_64,
```

Comment:
Forgot to enable?

Reply:
No, this is intentional, see https://github.com/mendersoftware/mender-demo-artifact/pull/2/commits/dcd6cf47d459f446a4c65c33da2615aa14eca23b.